README.md

```diff
@@ -19,6 +19,8 @@ to allow developers to setup a build and test environment in a few minutes.
 * Run tests on the device emulator ([Speculos](https://github.com/LedgerHQ/speculos)) or on a real device.
```

Comment:
Can you add a link to the tests section here ? 

Run [tests](https://developers.ledger.com/docs/device-app/integration/requirements/development#tests)

Reply:
Not part of this PR, but yes, of course.
Done :wink: